### DISCUSSION TITLE
[blocks]: Tree View

### DISCUSSION THREAD

**Discussion by ghost** [shadcn-ui/ui#3266] [Blocks Request]
### Description

A tree view that assembles all the functionalities of the Accordion component to create a tree view.

### Example

https://shadcn-extension-landing.vercel.app/docs/tree-view

**Comment by MildTomato** [comment]
[Another example](https://github.com/supabase/supabase/pull/21960) using react-accessible-treeview that could be ported to shadcn

https://github.com/supabase/supabase/assets/8291514/15274b70-0be2-4a01-8ebf-1fe6cdd395e1
